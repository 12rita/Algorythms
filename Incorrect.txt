log(tg(x)
lon(x)
x^cx
ln(2,x)
log(x)
