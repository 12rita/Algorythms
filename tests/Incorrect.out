incorrect input
incorrect input
error!
error!
incorrect input
